cms/core/blocks/markup.py

```diff
@@ -114,6 +114,11 @@ class ONSTableBlock(TinyTableBlock):
     source = blocks.CharBlock(label="Source", required=False)
     footnotes = blocks.RichTextBlock(label="Footnotes", features=settings.RICH_TEXT_BASIC, required=False)
 
+    def __init__(self, *, local_blocks=None, search_index=True, **kwargs):
+        super().__init__(local_blocks=local_blocks, search_index=search_index, **kwargs)
+        # relabeled to match editorial stuff terminology
```

Comment:
For the comment maybe something like: 
```suggestion
        # relabeled to match the publishing team's terminology
```
<br>
Also - the functional tests will need to updated to reflect this.